### DISCUSSION TITLE
cn/2024/06/17/life-map-and-loneliness/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#164] [Announcements]
# cn/2024/06/17/life-map-and-loneliness/

https://hongtaoh.com/cn/2024/06/17/life-map-and-loneliness/

**Comment by li199-code** [comment]
或许是之前一直走的是外界认为的好路，同时走的比较轻松。到了一定年岁，就突然发觉自己没有好好想过自己真正想要的是啥。

**Comment by Finderaha** [comment]
前段时间，一个经常玩的朋友离开了北京，去了南京，生活圈子忽然窄了很多。
北京太大，虽然同在一个城市，能见面的机会也是以月计量。
毕业之后，朋友们有了各自的人生轨迹，再也找不到之前那种志同道合的感觉了。
或许这是成长的代价，也或者这才是人生的常态。

我们的祖先将 群居/社群 的基因写进我们身体里，现在社会分工（或许是别的什么因素）又将群体拆解成个体。生活在当世，渴望集体又不可得 或许就是属于我们的“西西弗故事”，也是大多数人一生的宿命。

不是悲观哈，即是事实，那就事实好了。

我觉得 寻找伙伴最有效的方式是 做自己认可的、对别人有价值的事。因事而聚，虽然时间不久，但至少短期是能真诚相处的。

**Reply by hongtaoh** [reply]
我的感觉是不用去抗拒这种孤独，但是还是可以多出去走走，多认识一些人。
